rollup.config.js

```diff
@@ -61,6 +61,11 @@ const rollup = {
 		lightning({
 			options: {
 				include: Features.Nesting,
+				targets: {
+					ios_saf: 17,
+					chrome: 116,
+					firefox: 126,
+				},
```

Comment:
Bugfix for https://github.com/parcel-bundler/lightningcss/issues/826#issuecomment-2525433460


### Before
<img width="747" alt="Screenshot 3" src="https://github.com/user-attachments/assets/3ee3b203-3de8-4fe7-95f4-0ebfd647a390">

### After

<img width="515" alt="Screenshot 2" src="https://github.com/user-attachments/assets/0851b66d-bee9-4f67-bc00-bf132b7521d3">